### ISSUE TITLE
Camera: Failed to execute 'createObjectURL' on 'URL'

### ISSUE THREAD

**Issue by beevelop** [node-red-contrib-utils/node-red-contrib-browser-utils#15]
- Chrome: `Version 72.0.3626.28 (Official Build) beta (64-bit)`
- Node-RED: `0.19.5`
- node-red-contrib-browser-utils@`0.0.7`

## Exception
```
Failed to execute 'createObjectURL' on 'URL': No function was found that matched the signature provided
```

Almighty Stackoverflow suggests the following explanation:
> This error is caused because the function createObjectURL is deprecated for Google Chrome
The following snippet might be a suitable workaround / fix (in the `onStreamReady` function):
```js
video.srcObject=stream;
video.play();
```

## References
- https://stackoverflow.com/questions/27120757/failed-to-execute-createobjecturl-on-url

**Comment by chughts** [comment]
This affects the camera node only. Looks like createObjectURL is dropped as on Chrome 71. 

I am currently on 70.0.3538.102. 

I need a volunteer on 71 to test the fail, and code the updates. I can test if any code changes break the camera on Chrome 70

**Comment by chughts** [comment]
I am now on 71.0.3578.98 and getting this error. I can attempt to apply the fix, but will need someone on an earlier version to test and verify that it doesn't cause a regression.

**Comment by chughts** [comment]
Fixed in 0.0.8
